mix.exs

```diff
@@ -41,11 +41,18 @@ defmodule MixTestInteractive.MixProject do
 
   defp docs do
     [
-      extras: ["README.md"],
+      extras: [
+        "CHANGELOG.md": [],
+        "LICENSE.md": [title: "License"],
+        "README.md": [title: "Overview"],
+      ],
+      main: "readme",
+      source_url: @source_url,
```

Comment:
It can be put as project's property or within as docs' property. Since it's related to `mix docs`, it's preferable to group it together with `source_ref`.